test/integration/gardenlet/backupentry/backupentry/backupentry_suite_test.go

```diff
@@ -140,7 +140,7 @@ var _ = BeforeSuite(func() {
 		},
 	}
 
-	Expect(testClient.Create(ctx, gardenNamespace)).To(Or(Succeed(), BeAlreadyExistsError()))
```

Comment:
Why is this changed? Shouldn't we tolerate this error in case the integration test runs against an existing cluster?

I have to admit, even this is odd because we delete the namespace again after test execution. However, we follow this logic in several integration tests and should stick to either of the approaches.

Reply:
From https://github.com/gardener/gardener/blob/master/docs/development/testing.md#writing-integration-tests,
> don't tolerate already existing resources (~dirty test environment), code smell: ignoring already exist errors

>  However, we follow this logic in several integration tests and should stick to either of the approaches.

I agree, we should streamline this.

Reply:
Opened https://github.com/gardener/gardener/pull/7329